Include/Acidanthera/Library/OcBootManagementLib.h

```diff
@@ -2075,6 +2078,37 @@ OcAddEntriesFromBootEntryProtocol (
   IN     BOOLEAN        CreateForHotKey
   );
 
+/**
+  Useful internal vars from firmware UI protocol (cMP 144.0.0.0.0 ROM).
+**/
+typedef PACKED struct {
```

Comment:
This is still packed. Is it resolved or not?

Reply:
Oh for crying out loud, I'm editing while knackered and now ill 🤢 - very sorry. No it is wrong, thank you for rechecking. (I removed one of the two ways of specifying packed, only. I'll remove this one too, and write some debug code to retest before pushing again.)